packages/elasticsearch-plugin/src/options.ts

```diff
@@ -175,7 +175,8 @@ export interface ElasticsearchOptions {
      * parsed to the elasticsearch index.
      *
      * This config option defines custom mappings which are accessible when the "groupByProduct"
-     * input options is set to `true`.
+     * input options is set to `true`. Anyway, customProductVariantMappings can be accessed by using
+     * customProductVariantMappings field.
```

Comment:
>input options is set to `true`. Anyway, customProductVariantMappings can be accessed by using
customProductVariantMappings field.

change to 

"input options is set to `true`. In addition, custom variant mappings can be accessed by using
the `customProductVariantMappings` field, which is always available."